packages/uploadthing/src/internal/dev-hook.ts

```diff
@@ -28,7 +28,16 @@ export const conditionalDevServer = async (opts: {
       sdkVersion: UPLOADTHING_VERSION,
       fetch: opts.fetch,
     },
-    async (json: { fileData: FileData }) => {
+    async (json: {
+      fileData: {
+        callbackUrl: string;
+        callbackSlug: string;
+        metadata: string | null;
+        fileName: string;
+        fileSize: number;
+        customId: string | null;
+      };
+    }) => {
```

Comment:
decided to inline this - not used anywhere else and very specific to the callback simulation.